cli/config/config.go

```diff
@@ -23,6 +25,23 @@ type Config struct {
 	Clusters []*Cluster `toml:"cluster"`
 }
 
+// copied from cli/main.go
+func homedir() string {
```

Comment:
I don't think so, the new cross-compilation setup should allow it to work. Just make sure you test with a cross-compiled binary and it works.